docs/src/pages/components/DateField/datefield.example.html

```diff
@@ -0,0 +1,114 @@
+        <div class="ds-l-col--auto">
+          <label class="ds-c-label ds-c-datefield__label" for="year"><span>Year</span></label>
+          <input
+            class="ds-c-field ds-c-field--error ds-c-field--year"
+            type="text"
+            inputmode="numeric"
+            pattern="[0-9]*"
+            id="year-1"
+            name="birthdate[year]"
+            autocomplete="bday-year"
+            aria-describedby="datefield_label_1"
+            value="2050"
+          />
+        </div>
+      </div>
+      <span class="ds-c-field__error-message" role="alert">
+        Please enter a year in the past
+      </span>
+    </fieldset>
+    <div class="example--wrapper example--inverse">
+      <fieldset class="ds-c-fieldset">
+        <legend class="ds-c-label" id="datefield_label_2">
+          <span>Inverse example</span>
+          <span class="ds-c-field__hint ds-c-field__hint--inverse">For example: 4 / 28 / 1986</span>
+        </legend>
+        <div class="ds-l-form-row ds-u-align-items--end">
+          <div class="ds-l-col--auto">
+            <label class="ds-c-label ds-c-datefield__label" for="month"><span>Month</span></label>
+            <input
+              class="ds-c-field ds-c-field--month"
+              type="text"
+              inputmode="numeric"
+              pattern="[0-9]*"
+              id="month-2"
+              name="birthdate[month]"
+              autocomplete="bday-month"
+              aria-describedby="datefield_label_2"
+              value="10"
+            />
+          </div>
+          <span class="ds-c-datefield__separator">/</span>
+          <div class="ds-l-col--auto">
+            <label class="ds-c-label ds-c-datefield__label" for="day"><span>Day</span></label>
+            <input
+              class="ds-c-field ds-c-field--day"
+              type="text"
+              inputmode="numeric"
+              pattern="[0-9]*"
+              id="day-2"
+              name="birthdate[day]"
+              autocomplete="bday-day"
+              aria-describedby="datefield_label_2"
+              value="31"
+            />
+          </div>
+          <span class="ds-c-datefield__separator">/</span>
+          <div class="ds-l-col--auto">
+            <label class="ds-c-label ds-c-datefield__label" for="year"><span>Year</span></label>
+            <input
+              class="ds-c-field ds-c-field--error ds-c-field--inverse ds-c-field--year"
+              type="text"
+              inputmode="numeric"
+              pattern="[0-9]*"
+              id="year-2"
+              name="birthdate[year]"
+              autocomplete="bday-year"
+              aria-describedby="datefield_label_2"
+              value="2050"
+            />
+          </div>
+        </div>
+        <span class="ds-c-field__error-message ds-c-field__error-message--inverse" role="alert">
+            Please enter a year in the past
+          </span>
+      </fieldset>
+    </div>
+  </div>
```

Comment:
Add a newline to end of this file
```suggestion
  </div>
  
```